client/signup/steps/woocommerce-install/hooks/use-woop-handling/index.ts

```diff
@@ -141,8 +147,9 @@ export default function useEligibility( siteId: number ): EligibilityHook {
 		stagingDomain,
 		wpcomSubdomainWarning,
 
-		transferringBlockers,
-		hasBlockers: !! transferringBlockers.length,
+		transferringBlockers: transferringBlockers || [],
+		hasBlockers,
 		siteUpgrading,
+		isReadyForTransfer: ! hasBlockers && ! ( eligibilityWarnings && eligibilityWarnings.length ),
```

Comment:
I wonder if taking this approach could cause issues in the future. We'll be reporting that the site has blockers, but then not returning any. 

We could check here whether we as still fetching the eligibility data, and set `isReadyForTransfer` to `false` or `undefined` until we know either way.

It's perhaps not an issue and it depends on whether we're happy with boolean properties sometimes being undefined.

Reply:
addressed here b7b87931f30e8f97289443687cfdb0c5ccdc42af

Basically, it checks whether the eligibility data is available (not undefined), and sets the isReadyForTransfer considering this value.

Reply:
nice one, btw 💯